packages/client/src/components/Layouts/TopicLayout.tsx

```diff
@@ -0,0 +1,42 @@
+import { FormTitle, Heading } from "@amsterdam/asc-ui";
+import React from "react";
+
+import { HideForPrint } from "../../atoms";
+import { DebugVariables } from "../../debug";
+import { useTopic } from "../../hooks";
+import BaseLayout from "./BaseLayout";
+
+interface DefaultLayoutProps {
+  heading?: string;
+  formTitle?: string;
+}
+
+const DefaultLayout: React.FC<DefaultLayoutProps> = ({
```

Comment:
Please rename DefaultLayout to TopicLayout

Reply:
Good catch